docs/usage.md

```diff
@@ -69,119 +114,238 @@ cat ./compilation.json
+Once the build is logged successfully (and the `compile_commands.json`) was
+created, you can analyze your project.
 
-1. Run the analysis:
+### Run the analysis<a name="run-the-analysis"></a>
 ```sh
-CodeChecker analyze compilation.json -o ./reports
+CodeChecker analyze --output ./reports compile_commands.json
 ```
-2. View the analysis results in the command line
+
+However the compilation of the project (here the example program) is performed
+by `g++`, the CodeChecker uses `clang` to analyze sources of the project.
+During analysis, CodeChecker compiles/analyses sources again. The analysis
+process generally uses more time.
+
+The `./reports` directory is the "database" of CodeChecker that allows to
+manage further working steps.
+
+### View the analysis results in the command line<a name="view-analysis-results"></a>
 ```sh
 CodeChecker parse ./reports
 ```
 
 Hint:
- You can do the 1st and the 2nd step in one round by executing `check`
+You can do the 1st and the 2nd step in one round by executing `check`
 ```sh
-cd tmux
+cd <repo root dir>
+cd docs/examples
 make clean
-CodeChecker check -b "make" -o ./reports
+rm --recursive --force ./reports
+CodeChecker check --build "make" --output ./reports
 ```
-or to run on 22 threads
+or to run on 22 threads both the compilation and the analysis:
+
 ```sh
-CodeChecker check -j22 -b "make clean;make -j22" -o ./reports
+CodeChecker check --jobs 22 --build "make clean ; make --jobs 22" \
+    --output ./reports
 ```
 
-### Cross-Compilation <a name="cross-compilation"></a>
+### Cross-Compilation<a name="cross-compilation"></a>
 Cross-compilers are auto-detected by CodeChecker, so the `--target` and the
-compiler pre-configured
-include paths of `gcc/g++` are automatically passed to `clang` when analyzing.
+compiler pre-configured include paths of `gcc/g++` are automatically passed to
+`clang` when analyzing.
 
 **Make sure that the compilers used for building the project (e.g.
 `/usr/bin/gcc`) are accessible when `CodeChecker analyze` or `check` is
 invoked.**
 
-### Incremental Analysis <a name="incremental-analysis"></a>
+## Step 3: View analysis results in command line or generate static HTML files<a name="step-3"></a>
+You can print detailed results (including the control flow) in command line by
+running:
+
+```sh
+CodeChecker parse --print-steps ./reports
+...
+TODO: Cut the output of example prog. analysis here.
+...
+```
+
+It is possible to generate reports as plain `HTML` files using the
+`CodeChecker parse` command. (For other output formats please consult with help
+of `parse` subcommand.)
+
+```sh
+CodeChecker parse --export html --output ./reports_html ./reports
+...
+To view the results in a browser run:
+> firefox ./reports_html/index.html &
+```
+
+`./reports_html` directory will contain an `index.html` with a link to all
+findings that are stored in separate `HTML` files (one per analyzed build
+action).
+
+TODO: Change image following new example program.
```

Comment:
Handle all the TODO's which you introduced in this PR.

Reply:
Done.